clamav-1.4-docker.yaml

```
@@ -0,0 +1,41 @@
+package:
+  name: clamav-1.4-docker
+  version: "1.4.2"
+  epoch: 0
+  description: An anti-virus toolkit for UNIX eis-ng backport
+  copyright:
+    - license: GPL-2.0-only
+  dependencies:
+    runtime:
+      - clamav-1.4
+      - libstdc++
+      - tini
+      - tzdata
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/Cisco-Talos/clamav-docker
+      expected-commit: 658ba84dfb16f0a70ea6588c5da208e36962e678
+      tag: ""
+
+  - runs: |
+      addgroup -S "clamav"
+      adduser -D -G "clamav" -h "/var/lib/clamav" -s "/bin/false" -u 100 -S "clamav"
+
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      mv /home/build/clamav/1.4/alpine/scripts/docker-entrypoint-unprivileged.sh "${{targets.destdir}}"/usr/bin/init-unprivileged
+      mv /home/build/clamav/1.4/alpine/scripts/docker-entrypoint.sh "${{targets.destdir}}"/usr/bin/init
+      mv /home/build/clamav/1.4/alpine/scripts/clamdcheck.sh "${{targets.destdir}}"/usr/bin/clamdcheck.sh
+
+      chmod -R 755 "${{targets.destdir}}"/usr/bin/
+      chown -R clamav:clamav "${{targets.destdir}}"/usr/bin/
+
+update:
+  enabled: false
```

we can poll it weekly using git backend. 💭 
```suggestion
update:
  enabled: true
  git: {}
  schedule:
    period: weekly
    reason: Upstream does not maintain tags or releases
```